.gitignore

```diff
@@ -1 +1,9 @@
+# Project
 devctl*
+bin-dist/
+TODO
+
+# OS & IDE
+.DS_Store
+.idea/
+.vscode/
```

Comment:
IMO you should have this stuff in your global gitignore

Reply:
I already thought about something like this, but someone else added them to other repositories, too. So: Is there a doc about this? At least I didn't find one, also took a look inside giantswarm/fmt.

Reply:
Not really, not a big deal.